ui/js/controllers/perf/compare.js

```diff
@@ -853,3 +864,92 @@ perf.controller('CompareSubtestResultsCtrl', [
             });
         });
     }]);
+
+perf.controller('CompareSubtestDistributionCtrl', ['$scope', '$stateParams', '$q', 'ThRepositoryModel',
+    'PhSeries', 'ThResultSetModel', 'metricsgraphics',
+    function CompareSubtestDistributionCtrl($scope, $stateParams, $q, ThRepositoryModel,
+        PhSeries, ThResultSetModel, metricsgraphics) {
+        $scope.originalRevision = $stateParams.originalRevision;
+        $scope.newRevision = $stateParams.newRevision;
+        $scope.originalSubtestSignature = $stateParams.originalSubtestSignature;
+        $scope.newSubtestSignature = $stateParams.newSubtestSignature;
+        $scope.dataLoading = true;
+        let loadRepositories = ThRepositoryModel.load();
+        const drawReplicateGraph = function (project, revision, subtestSignature, target) {
+            //let resultSet;
+            let replicateData = {};
+            return ThResultSetModel.getResultSetsFromRevision(project, revision).then(
+                (revisionData) => {
+                    replicateData.resultSet = revisionData[0];
+                    return PhSeries.getSeriesData(project, {
+                        signatures: subtestSignature,
+                        push_id: replicateData.resultSet.id
+                    });
+                }).then((perfDatumList) => {
+                    if (!perfDatumList[subtestSignature])
+                        return;
+                    let noOfRuns = perfDatumList[subtestSignature].length;
+                    let replicatePromises = perfDatumList[subtestSignature].map(
+                        (value) => {return PhSeries.getReplicateData({job_id: value.job_id});
+                        });
+                    return $q.all(replicatePromises).then((replicateData) => {
+                        let replicateValues = replicateData.concat.apply([],
+                                replicateData.map(function (data) {
+                                    let testSuite = data.suites.find(function (suite) {
+                                        return suite.name === $scope.testSuite;
+                                    });
+                                    let subtest = testSuite.subtests.find(function (subtest) {
+                                        return subtest.name === $scope.subtest;
+                                    });
+                                    return subtest.replicates;
+                                })
+                            );
+                        replicateValues = replicateValues.map((value, index) => {
+                            return {"Replicate": (index + 1).toString(), "Value": value};
+                        });
+                        metricsgraphics.data_graphic({
+                            title: target + " Repicates: " + noOfRuns + " Test " + (noOfRuns === 1? "Run":"Runs"),
```

Comment:
Repicates -> Replicates. 

Also this would be better done with a javascript string literal. e.g.:
```js
metricsgraphics.data_graphic({
  title: `${target} Replicates over ${numRuns} run${(numRuns > 1) ? 's' : ''}`,
```